app/models.py

```diff
@@ -148,6 +148,7 @@ class Carpool(db.Model, UuidMixin):
     destination_id = db.Column(db.Integer, db.ForeignKey('destinations.id'))
 
     ride_requests = relationship("RideRequest", cascade="all, delete-orphan")
+    driver = relationship("Person", foreign_keys=[driver_id])
```

Comment:
I think you might be able to get away with the `foreign_keys` specified here?

Reply:
Yep, for some reason I imagined that this model had multiple relationships to `Person` 😅. Amended.